### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by MarkoKermi**
In this PR me and @zewdu444 manage to do the following requirements:

- [ ] We instal and setup the devise:

- Can register a new user.
- User logs in with a combination of email and password.
- Hashed password should be stored in the database.
- Ask for confirmation of email.
- Can reset password. 

- [ ] Add some style.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,7 +1,11 @@
 class User < ApplicationRecord
-  has_many :comments, foreign_key: :author_id
-  has_many :posts, foreign_key: :author_id
-  has_many :likes, foreign_key: :author_id
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```
Comment (KDlamini) [inline app/models/user.rb]: Great job installing and setting up user authentication with Devise! 👏🏾💯 

As per project requirements kindly make sure that user registration asks for email confirmation.

image

- You can achieve this by uncommenting ` # :confirmable` in the User model.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi team @MarkoKermi, @zewdu444  🙋‍♀️,  

Great work so far! 💯 
You have done a great job adding user authentication using Devise in your Rails Blog App. 🎊 🥳 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 

✔️  Professional pull request
✔️  Well documented README 
✔️  The application allows for registering and logging in as a new user
✔️   A user can reset the password in the application.

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/user.rb (hunk 1 above)
Comment (MarkoKermi) [reply app/models/user.rb]: @KDlamini thx for the suggestion and review!

**Review by DammyShittu — APPROVED** [review]
Hi @MarkoKermi and @zewdu444,

## APPROVED 🟢

## Highlights
✔️ Devise setup

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**